admin/src/translations/es.json

```diff
@@ -0,0 +1,48 @@
+{
+    "plugin.name": "Calendario",
+    "plugin.tagline": "Visualice sus eventos",
+    "view.calendar.state.empty.configure-settings.message": "Configure los ajustes antes de acceder al calendario",
+    "view.calendar.state.empty.configure-settings.action": "Ajustes",
+    "view.calendar.action.create-entry": "Crear nueva {colección}",
+    "view.calendar.action.select-date": "Seleccionar fecha",
+    "view.calendar.action.today": "Hoy",
+    "view.calendar.action.month": "Mes",
+    "view.calendar.action.week": "Semana",
+    "view.calendar.action.day": "Día",
+    "view.calendar.locale": "es-ES",
+    "view.calendar.first-day-of-week": "0",
+    "view.settings.title": "Ajustes del calendario",
+    "view.settings.subtitle": "Configure el complemento como mejor se adapte a sus necesidades",
+    "view.settings.action.save": "Guardar",
+    "view.settings.section.general.title": "Configuración general",
+    "view.settings.section.general.collection.label": "Elija su colección",
+    "view.settings.section.general.title.label": "Elija su campo de título",
+    "view.settings.section.general.title.none": "Sin campo de título",
+    "view.settings.section.general.default-duration.label": "Elija la duración predeterminada de su evento",
+    "view.settings.section.general.default-duration.30min": "30 minutos",
+    "view.settings.section.general.default-duration.1h": "1 hora",
+    "view.settings.section.general.default-duration.1.5h": "1,5 horas",
```

Comment:
can we use a period `.` instead of a comma?

Reply:
Both are acceptable, but the comma has been the norm in Spanish for longer, so it is used in more Spanish-speaking countries.  I would recommend a comma, unless the target audience is largely from Spanish-speaking countries which have adopted the English-inspired decimal point instead of the decimal comma, but both options are acceptable.

There's a handy world map of decimal separator preferences on [the Wikipedia decimal separator page](https://en.wikipedia.org/wiki/Decimal_separator). And [here](https://www.rae.es/dpd/coma) you are the official line on it from the Royal Spanish Academy.

Reply:
_Very_ interesting. Commas are fine. 
Thanks for clarifying that